tools/ops/policylambda.py

```diff
@@ -135,6 +136,13 @@ def dispatch_render(p, sam):
         return None
     policy_lambda = mu.PolicyLambda(p)
     properties = render_func(p, policy_lambda, sam)
+    role_arn_pattern = re.compile("^arn:(aws[a-zA-Z-]*)?:iam::(\d{12}|{account_id}):role\/?[a-zA-Z_0-9+=,.@\-_\/]+$")
```

Comment:
I think the simple thing of just matching is `if properties['Role'].startswith('arn:')` is sufficient here.